{{cookiecutter.project_slug}}/clients/mobile/react-native/src/services/user/api.ts

```diff
@@ -37,7 +38,7 @@ const resetPassword = createCustomServiceCall(
 export const userApi = createApi(
   {
     client: axiosInstance,
-    baseUri: '/api/users/',
+    baseUri: 'api/users/',
```

Comment:
fun

Reply:
Hmm I don't think that slash matters given that the axios instance should already have the baseUrl defined?, so whether or not you add the slash, axios will know whether it should add it or not

Reply:
I was just observing that when I've seen changes like this in the past, it's because things are fragile with slashes missing or extra ones sneaking in and it being easy for someone to break.
Ideally we have a URL builder or a utility that sanitizes URLs before we use them. This exists on the BE in Python as part of the standard library. Not sure if we have it on the frontend.

Not relevant for this PR. Just a code smell.

Reply:
Well actually @lakardion built tn-models to handle that (at least the  trailing slash) right?

Reply:
trailing is handled.
leading, imo doesn't matter. ran a quick test and the request is the same whether you do `api` or `/api` with axios

Reply:
nice!